packages/tables/src/Table.php

```diff
@@ -339,6 +339,17 @@ public function getRecordUrl(Model $record): ?string
         return $callback($record);
     }
 
+    public function isRecordSelectable(Model $record): bool
+    {
+
+        /** @var TableComponent $livewire */
+        $livewire = $this->getLivewire();
+
+        $callback = $livewire->getTableRecordBulkActionsEnabled();
+
+        return $callback ? $callback($record) : true;
+    }
```

Comment:
To follow the style of the other code:

```suggestion
    public function isRecordSelectable(Model $record): bool
    {
        /** @var TableComponent $livewire */
        $livewire = $this->getLivewire();

        $callback = $livewire->getTableRecordBulkActionsEnabled();
        
        if (! $callback) {
            return true;
        }

        return $callback($record);
    }
```